### PR TITLE
Remove load of guardian configuration from sdl-tool-cfg

### PR DESCRIPTION
**Pull request by jakubstilec**
https://github.com/dotnet/core-eng/issues/11902

### CODE REVIEW

File: eng/common/sdl/init-sdl.ps1 [hunk 1]
```diff
@@ -29,18 +29,19 @@ $zipFile = "$WorkingDirectory/gdn.zip"
...
-  Invoke-WebRequest -Headers @{ "Accept"="application/zip"; "Authorization"="Basic $encodedPat" } -Uri $uri -OutFile $zipFile
-  if (Test-Path $gdnFolder) {
-    # Remove the gdn folder if it exists (it shouldn't unless there's too much caching; this is just in case)
-    Remove-Item -Force -Recurse $gdnFolder
-  }
-  [System.IO.Compression.ZipFile]::ExtractToDirectory($zipFile, $WorkingDirectory)
-  Write-Host $gdnFolder
-  ExitWithExitCode 0
-} catch [System.Net.WebException] { } # Catch and ignore webexception
+# removed load of guardian configuration from sdl-tool-cfg
```
Comment (adiaaida) [inline eng/common/sdl/init-sdl.ps1:32]: Should we just delete these lines instead of commenting them out?

Comment (jakubstilec) [reply eng/common/sdl/init-sdl.ps1]: good point, I wanted to leave it as it gives you hint in a case of SDL issues, but it's better to cleanup unused code

**Review by adiaaida — APPROVED** [review]

**Comment by dagood** [comment]
Is this good to merge? 🙂 I'm waiting on a new dotnet/arcade official build and this seems to be needed to unblock the pipeline.

**Review by MattGal — APPROVED** [review]

**Comment by dagood** [comment]
Thanks! Build went through.

**Comment by jakubstilec** [comment]
Great to hear @dagood!

**Pull request merged**